waku/common/wakubridge.nim

```diff
@@ -91,7 +91,7 @@ func toWakuMessage(env: Envelope): WakuMessage =
   # Translate a Waku v1 envelope to a Waku v2 message
   WakuMessage(payload: env.data,
               contentTopic: toV2ContentTopic(env.topic),
-              timestamp: float64(env.expiry - env.ttl),
+              timestamp: int64(env.expiry - env.ttl),
```

Comment:
If `env.expiry` and `env.ttl` have a different resolution, then this needs to be converted to nanosecond